code/game/turfs/open/chasm.dm

```diff
@@ -75,7 +75,9 @@
 				to_chat(user, "<span class='warning'>You need one floor tile to build a floor!</span>")
 		else
 			to_chat(user, "<span class='warning'>The plating is going to need some support! Place metal rods first.</span>")
-
+/turf/open/chasm/CanAllowThrough(atom/movable/mover, turf/target)
+	SHOULD_CALL_PARENT(FALSE)
+	return TRUE
```

Comment:
This is not a proper solution, we want to preserve the parent call here.
The logic needs to be adapted for it.